src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1841,11 +1841,24 @@ public function inline_edit() {
 					<?php foreach ( $flat_taxonomies as $taxonomy ) : ?>
 
 						<?php if ( current_user_can( $taxonomy->cap->assign_terms ) ) : ?>
-							<?php $taxonomy_name = esc_attr( $taxonomy->name ); ?>
+							<?php
+							$taxonomy_name = esc_attr( $taxonomy->name );
+
+							$tags = get_tags(
+								array(
+									'fields' => 'names',
```

Comment:
Adding this solve [my comment](https://github.com/ClassicPress/ClassicPress/pull/1287#issuecomment-1874038714).
`'hide_empty' => false,` 

Reply:
Now added